shared/reducers/fs.js

```diff
@@ -292,6 +297,13 @@ export default function(state: Types.State = initialState, action: FsGen.Actions
     case FsGen.setSendLinkToChatChannels:
       // $FlowIssue
       return state.setIn(['sendLinkToChat', 'channels'], action.payload.channels)
+    case FsGen.setSendAttachmentToChatConvID:
+      // $FlowIssue
```

Comment:
why flow issue

Reply:
I think `setIn` with flow is just broken. I'll see if I can get around it with other mutaters.